easybuild/easyconfigs/r/R/R-3.2.0-goolf-1.7.20.eb

```diff
@@ -0,0 +1,430 @@
+name = 'R'
+version = '3.2.0'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no --enable-R-shlib"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('libpng', '1.6.17'),  # for plotting in R
+    ('libjpeg-turbo', '1.4.0'),  # for plottting in R
+    ('Java', '1.8.0_25', '', True),  # Java bindings are built if Java is found, might as well provide it
```

Comment:
This is a never-ending story... The only true solution here is to support something like:

```python
('Java', '>=1.8', '', True)
```

which is probably a nice can of worms by itself (think reproducibility of builds, both on the same system/by the same user, or on a different system/by someone else)